### PR TITLE
Update CENTRIFUGE: bump versions, fix missing args, and command formatting

### PR DESCRIPTION
**Pull request by jfy133**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [ ] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

File: modules/nf-core/centrifuge/centrifuge/main.nf [hunk 1]
```diff
@@ -14,26 +14,27 @@ process CENTRIFUGE_CENTRIFUGE {
...
-    tuple val(meta), path('*.{sam,tab}')                 , optional: true, emit: sam
-    tuple val(meta), path('*.mapped.fastq{,.1,.2}.gz')   , optional: true, emit: fastq_mapped
-    tuple val(meta), path('*.unmapped.fastq{,.1,.2}.gz') , optional: true, emit: fastq_unmapped
-    path "versions.yml"                                  , emit: versions
+    tuple val(meta), path('*report.txt'), emit: report
+    tuple val(meta), path('*results.txt'), emit: results
+    tuple val(meta), path('*.{sam,tab}'), optional: true, emit: sam
+    tuple val(meta), path('*.mapped.fastq{,.1,.2}.gz'), optional: true, emit: fastq_mapped
+    tuple val(meta), path('*.unmapped.fastq{,.1,.2}.gz'), optional: true, emit: fastq_unmapped
+    path "versions.yml", emit: versions
```
Comment (LilyAnderssonLee) [inline modules/nf-core/centrifuge/centrifuge/main.nf:22]: Why did you remove the space alignment here?

**Review by LilyAnderssonLee — DISMISSED** [review]

File: modules/nf-core/centrifuge/build/main.nf [hunk 2]
```diff
@@ -44,7 +46,7 @@ process CENTRIFUGE_BUILD {
     """
 
     stub:
-    def args = task.ext.args ?: ''
+    def _args = task.ext.args ?: ''
```
[the remark this answers is not in the document]
Comment (jfy133) [reply modules/nf-core/centrifuge/build/main.nf:49]: It's the Language Server/Nextflow VSCode extension way of referring to variables that don't get used (you cans ee this in the warnings section)

[the remark this answers is not in the document]
Comment (sateeshperi) [reply modules/nf-core/centrifuge/build/main.nf:49]: ya but even better to just remove unused ones ;)

**Review by sateeshperi — DISMISSED** [review]

File: tests/config/nf-test.config [hunk 3]
```diff
@@ -29,7 +29,7 @@ profiles {
         docker.enabled          = true
         docker.userEmulation    = false
         docker.fixOwnership     = true
-        docker.runOptions       = '--platform=linux/amd64 --security-opt=no-new-privileges'
+        docker.runOptions       = '-u $(id -u):$(id -g) --platform=linux/amd64  --security-opt=no-new-privileges --userns=host'
```
Comment (sateeshperi) [inline tests/config/nf-test.config:32]:
```suggestion
        docker.runOptions       = '-u $(id -u):$(id -g) --platform=linux/amd64'
```

**Review by matthdsm — APPROVED** [review]

File: tests/config/nf-test.config (hunk 3 above)
Comment (sateeshperi) [reply tests/config/nf-test.config:32]: lets get it how it was before as these options obviously didn't fix the issue

**Review by erikrikarddaniel — DISMISSED** [review]
:+1:

**Review by sateeshperi — APPROVED** [review]
